CoreFindView/src/au/gov/asd/tac/constellation/views/find/FindViewTopComponent.java

```diff
@@ -93,8 +91,7 @@ public FindViewTopComponent() {
          * default to avoid index out of bounds issues when trying to find a node or transaction that no longer exists.
          */
         addStructureChangeHandler(graph -> {
```

Comment:
You could probably remove the curly braces here. So it would be like `addStructureChangeHandler(graph -> FindViewController.getDefault().clearResultsLists());` instead